himin/day6/이상한 문자 만들기.js

```diff
@@ -0,0 +1,13 @@
+function solution(s) {
+    return s
+        .split(" ")
+        .map((word) =>
+            word
+                .split("")
+                .map((char, index) =>
+                    index % 2 === 0 ? char.toUpperCase() : char.toLowerCase()
+                )
+                .join("") 
+        )
+        .join(" "); 
+}
```

Comment:
```suggestion
function solution(s) {
  return s
    .split(' ')
    .map(word => changeText(word))
    .join(' ');
}

function changeText(word) {
  return word
    .split('')
    .map((char, index) =>
      index % 2 === 0 ? char.toUpperCase() : char.toLowerCase()
    )
    .join('');
}
```
메서드 체이닝은 사용할때는 좋지만, 가독성이 좋지 않다는 단점이 있습니다🥲
메서드 내부에서 사용되는 함수를 분리해서 관리하면 가독성이 올라갈 것 같습니다😊

Reply:
@zzzRYT 오 좋은 방법 감사합니다!